lib/mix/tasks/token.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Mix.Tasks.Cadet.Token do
+  @moduledoc """
+  Helper to generate access_token to ease development.
+
+  Usage:
+    mix task cadet.token <role>
+  """
+
+  @shortdoc "Generates access_token JWT for a given user role"
+
+  use Mix.Task
+
+  import Mix.Ecto
+  import Ecto.Query
+
+  alias Cadet.Accounts.{Role, User}
+  alias Cadet.Auth.Guardian
+  alias Cadet.Repo
+
+  @env_allow_mock ~w(dev)a
+
+  def run(args) do
+    # Suppress Ecto SQL logs
+    Logger.configure(level: :error)
+
+    # Required for Ecto to work properly, from Mix.Ecto
+    ensure_started(Repo, [])
+
+    roles = Enum.filter(Role.__valid_values__(), &is_binary/1)
```

Comment:
Can replace with `roles = Role.__enum_map__()`

Reply:
That would be atoms tho.

I could also do `Enum.map(Role.__enum_map__(), &Atom.to_string/1)`

Reply:
derped. mb